releases/CHANGELOG-v0.79.0.md

```diff
@@ -0,0 +1,14 @@
+
+<a name="v0.79.0"></a>
+# [v0.79.0] - 2023-08-29
+### Features 🌈
+- add support for Kubernetes v1.24.17 ([#5112](https://github.com/Azure/aks-engine/issues/5112))
```

Comment:
we're missing the Azure CNI bump

Reply:
https://github.com/Azure/aks-engine/commit/3970026335e1ef85ef2c41b22164074a31d3da77

Reply:
I added it in the "Maintenance" section.